test/get_images/test_images.yaml

```diff
@@ -40,3 +40,7 @@ sources:
       unzip_xml: image.gen1.xml
   - btrfs:
       image: https://corp.digitalcorpora.org/corpora/drives/2015-basicmaster/btrfs_testimage_50MB.E01
+  - xfs:
+      zipfile: https://files.iblue.team/279b6e00-851e/2GB-xfs/2GB-xfs-raw.zip
+      unzip_image: 2GB-xfs-raw.001
+
```

Comment:
I'm really excited about this disk image. However, we will need to recode it as an E01.

Reply:
I'll take care of it later today. Thanks.

Reply:
I wonder why this file needs to be so big....

Reply:
```
-rw-r--r--@ 1 simsong  staff  2147483648 Aug 24  2023 2GB-xfs-raw.001
-rw-r--r--@ 1 simsong  staff     3958191 Dec 20 13:53 2GB-xfs-raw.E01
-rw-r--r--@ 1 simsong  staff     2538384 Dec 19 09:03 2GB-xfs-raw.zip
```
This image is now at https://corp.digitalcorpora.org/corpora/drives/tsk-2024/ and can be downloaded from: https://digitalcorpora.s3.amazonaws.com/corpora/drives/tsk-2024/2GB-xfs-raw.E01

Reply:
> Thanks, I'm using the new format and fixed all current issue

Great!